app/src/main/java/com/arygm/quickfix/ui/Announcement.kt

```diff
@@ -26,56 +26,46 @@ import com.arygm.quickfix.ui.navigation.Route
 @Composable
 fun AnnouncementScreen(navigationActions: NavigationActions, isUser: Boolean, LoD: Boolean) {
 
-    val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
-    val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
+  val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
+  val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
```

Comment:
The color1 and backgroundColor variables are declared twice. Move them outside the CalendarScreen function to define them globally, or place them in a separate theme configuration file if they are reused elsewhere in the app

Reply:
done :)